configs/envoy_double_proxy.template.yaml

```diff
@@ -0,0 +1,110 @@
+---
+{% macro listener(address,ssl,proxy_proto) -%}
+- address: "{{ address }}"
```

Comment:
this address needs to follow https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/address.proto#envoy-api-msg-core-address

Reply:
Needs to look akin to: 
```
- address:
    socket_address:
      address: {{ address }}
      protocol: {{ protocol }}
      port_value: {{ port }}
```